CMakeLists.txt

```diff
@@ -1,53 +1,115 @@
 cmake_minimum_required(VERSION 3.5.1)
 cmake_policy(SET CMP0069 NEW)
 
```

Comment:
this part might be annoying, if we could fetch that version number automatically somehow it would be nice otherwise each time a release is made the author should remember to change 6.0.5 into 6.0.6

Reply:
There is a `postversion` Makefile target that is currently used to push the artifacts to `release` branch. I suppose we could place something like `<<VERSION>>` here and `sed` it in the `CMakeLists.txt` before pushing to `release` branch.